docs/installation.rst

```diff
@@ -79,11 +79,14 @@ Installation of the Connectome Mapper 3 has been facilitated through the distrib
 
 *   Open a terminal
 
-*   Get the latest release (|release|) of the BIDS App:
+*   Download and extract the latest release (|release|) of the BIDS App:
 
 .. parsed-literal::
 
     $ docker pull sebastientourbier/connectomemapper-bidsapp:|release|
+    
+.. note:: This can take a little bit of time depending on your connection speed and your machine.
```

Comment:
I would say "some" instead of "a little bit"

Reply:
same below

Reply:
```suggestion
.. note:: This can take some time depending on your connection speed and your machine.
```